README.markdown

```diff
@@ -12,23 +12,6 @@ Easily inject arbitrary read-only resources into executable formats
 npm i -g postject
 ```
 
-## Building
```

Comment:
Heh, according to git my change was about moving the `Building` section below 😄

Reply:
It's git's world, we're all just living in it.